tests/test_reports.py

```diff
@@ -12,7 +12,8 @@ def test_request_report(mws_client, mock_adapter, example_response):
     response = mws_client.reports.request_report(
         Marketplace='ATVPDKIKX0DER',
```

Comment:
Do we have to pass `MarketplaceIdList` and `Marketplace` both param to the call ?

Reply:
No. Actually, only MarketPlaceIdList will be respected